common/adapters.ts

```diff
@@ -164,6 +164,7 @@ export const adapterSettings: { [key in keyof Preset]: AIAdapter[] | readonly AI
   oaiModel: ['openai', 'kobold'],
   frequencyPenalty: ['openai', 'kobold'],
   presencePenalty: ['openai', 'kobold'],
+  streamResponse: ['openai', 'kobold'],
```

Comment:
Though currently I only wrote a streaming implementation for OpenAI, I added `kobold` following the pattern for freq/presence penalties in case that was how to get it to properly support OAI-compatible third party APIs. Not sure if it's actually necessary, though.